bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -78,6 +80,32 @@ trait UtilBijections {
     }
   }
 
+  private val futureConverter = {
+    val converter = new JavaFutureToTwitterFutureConverter()
+    converter.start()
+    converter
+  }
+
+  /**
+   * Injection from java futures to twitter futures
+   */
+  implicit def twitter2JavaFuture[A]: Bijection[TwitterFuture[A], JavaFuture[A]] = {
```

Comment:
what if we take an `implicit JavaFutureToTwitterFutureConverter` here as a parameter, and then it is the responsibility of the user to create and eventually stop their futureConverter?